init.lua

```diff
@@ -8,3 +8,7 @@ for _, module in ipairs(init_modules) do
       error("Error loading " .. module .. "\n\n" .. err)
    end
 end
+
+-- This runs your userspace function within chadrc
+-- It will 'likely' run after all configuration (though not guaranteed)
+vim.cmd(' au VimEnter * lua require("chadrc").userspace()')
```

Comment:
Because the autocommand `VimEnter` will happen after NeoVim/NvChad has finished most of the startup & configuration.

Thus it reduces the possibility of users breaking things or breaking the setup of plugins by just setting an option part way through initialisation.